mesonbuild/backend/ninjabackend.py

```diff
@@ -1396,7 +1396,7 @@ def generate_dynamic_link_rules(self, outfile):
  rspfile_content = $ARGS  {output_args} $in $LINK_ARGS {cross_args} $aliasing
 '''
                 else:
-                    command_template = ' command = {executable} $ARGS {output_args} $in $LINK_ARGS {cross_args} $aliasing\n'
+                    command_template = ' command = {executable} $ARGS {cross_args} {output_args} $in $LINK_ARGS $aliasing\n'
```

Comment:
Please don't make such changes in an aside without rational. Argument ordering in one of those places that might or might not have consequences.